### PR TITLE
Story 22 GET /college

### PR DESCRIPTION
**Pull request by Korbrent**
It'll need to be changed when the CollegeUsers gets added, I'll go back and do that.

### CODE REVIEW

File: src/main/java/com/collegecompendium/backend/controllers/CollegeController.java [hunk 1]
```diff
@@ -82,6 +82,33 @@ public College createNewCollege(@RequestBody College input, @AuthenticationPrinc
...
+	public College getCollege(
+			@AuthenticationPrincipal Jwt token,
+			HttpServletResponse response) {
+
+		// This will have to change depending on how CollegeUser is implemented.
+		College result = collegeRepository.findDistinctByAuth0Id(token.getSubject());
+
+		// if the college doesn't exist then send back a 403
+		if (result == null) {
+			response.setStatus(403);
```
Comment (meta1203) [inline src/main/java/com/collegecompendium/backend/controllers/CollegeController.java]: 403 is "forbidden", this should be a 404 "not found"

**Review by meta1203 — COMMENTED** [review]
403 -> 404 and we're good to go

File: src/main/java/com/collegecompendium/backend/controllers/CollegeController.java (hunk 1 above)
Comment (Korbrent) [reply src/main/java/com/collegecompendium/backend/controllers/CollegeController.java]: Oops thank you

**Review by meta1203 — APPROVED** [review]
Looks good

**Pull request merged**